### ISSUE TITLE
Feature request: Variable expansion in .env

### ISSUE THREAD

**Issue by prsdta** [percy/cli#410]
Hello, I see Percy is using `dotenv` to read `.env` variables, in the rails way which is very useful.
However, the ruby `dotenv` implementation also does variable interpolation.
Example:
```
USER=postgres
DATABASE_TEST_URL="postgres://$USER:@localhost/db"
```

Could you add it, e.g. with https://github.com/motdotla/dotenv-expand ?

Thank you.

**Comment by Robdel12** [comment]
Closed with #426

**Comment by prsdta** [comment]
Thank you!
